definitions/ext-service/definition.yml

```diff
@@ -42,6 +42,18 @@ synthesis:
           entityTagName: language
         telemetry.sdk.version:
 
+    # telemetry from opentelemetry provider
+    - identifier: service.name
+      name: service.name
+      encodeIdentifierInGUID: true
+      conditions:
```

Comment:
will this one generate the same telemetry as the one above this rule?

Reply:
No, the tag translation happens after there is a match

Reply:
I do agree on that. But do we need to have different Golden Metrics and summary metrics for this two? if that's the case having the same instrumentation.provider will be an issue.

If we don't need to, this is fine as it is

Reply:
I don't think so because both are open telemetry so I'd expect the attributes to be the same (reinforcing this: we restored this rule because we noticed some entities missing, so the GMs were showing correctly already).